api/nodes/views.py

```diff
@@ -1,9 +1,9 @@
 import requests
 
 from modularodm import Q
+from rest_framework import status
 from rest_framework import generics, permissions as drf_permissions
 from rest_framework.exceptions import PermissionDenied, ValidationError, NotFound
-from rest_framework.status import is_server_error
```

Comment:
Why was this removed in favor of import the whole `status` module? It's not of great importance, but would likely incur a small performance hit.